starter-code/scripts/articles.js

```diff
@@ -27,6 +36,8 @@ Article.prototype.toHtml = function() {
   for that before this current article gets rendered to our
   DOM. */
 
+  $newArticle.removeAttr('class');
```

Comment:
It would be preferable to use `removeClass()` method, in case there are other classes present that you do not want to affect.